twake/backend/node/test/e2e/users/users.spec.ts

```diff
@@ -3,7 +3,8 @@ import { init, TestPlatform } from "../setup";
 import { TestUsers } from "./utils";
 
 describe("The /users API", () => {
-  const url = "/internal/services/users/v1/users";
+  jest.setTimeout(30000);
```

Comment:
Should be in JEST configuiration

Reply:
But we don't have any for the project. I added jest.config.js once, but Romaric said it could be the reason all tests failed  and I removed it

Reply:
If you look at "test:e2e" in package.json you will see ` --testTimeout=20000`, just add it to your npm command

Reply:
yep, but the problem is that I do TDD and run tests directly from IDE in debug mode (so I don't use package.json scripts for this), and Idea creates configurations for every test run automatically. but that's ok, I can handle it on my side. I've removed this line/